### ISSUE TITLE
Opening new tabs is slow over VPN because it reconnects to the server and reloads the list of tables

### ISSUE THREAD

**Issue by dsego** [jakob/Postico#869]
## What did you do?

- Clicked cmd+T to open a new tab.

## What did you expect to happen?

- I expected to quickly open a new tab to see data in a different table in the same database schema.

## What actually happened?

- The app opened a new tab but started from scratch, reconnecting to the same database and reloading all tables from scratch, which took several seconds over VPN.

## What software versions are you using?

Postico version:  2.1 dev
macOS version:  14.1.2
PostgreSQL version:  PostgreSQL 9.6.24

**Comment by jakob** [comment]
Yeah, tabs in Postico are completely independent. Open a new tab, and it shares no state with other tabs. Each tab has its own database connection, etc.

I understand that this isn't ideal for a lot of use cases, but changing it would be a major undertaking and come with some drawbacks -- eg. I'm not sure how to support simultaneous execution of queries when tabs share connections.

**Comment by dsego** [comment]
Thanks for the reply. I haven't found the issue reported, and it's something the is certainly not ideal for me, so I thought I'd post here. Certainly feel free to close this issue or tag appropriately  if it's something you're not considering in the near future.
